[2025-04-28T18:17:52.768+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gen_lxw_fts.mateiralize_fts scheduled__2025-03-07T23:00:00+00:00 [queued]>
[2025-04-28T18:17:52.782+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gen_lxw_fts.mateiralize_fts scheduled__2025-03-07T23:00:00+00:00 [queued]>
[2025-04-28T18:17:52.783+0000] {taskinstance.py:2171} INFO - Starting attempt 2 of 3
[2025-04-28T18:17:52.809+0000] {taskinstance.py:2192} INFO - Executing <Task(BashOperator): mateiralize_fts> on 2025-03-07 23:00:00+00:00
[2025-04-28T18:17:52.817+0000] {standard_task_runner.py:60} INFO - Started process 9248 to run task
[2025-04-28T18:17:52.822+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'gen_lxw_fts', 'mateiralize_fts', 'scheduled__2025-03-07T23:00:00+00:00', '--job-id', '285', '--raw', '--subdir', 'DAGS_FOLDER/gen_lxw_fts.py', '--cfg-path', '/tmp/tmpijgixrnz']
[2025-04-28T18:17:52.823+0000] {standard_task_runner.py:88} INFO - Job 285: Subtask mateiralize_fts
[2025-04-28T18:17:52.913+0000] {task_command.py:423} INFO - Running <TaskInstance: gen_lxw_fts.mateiralize_fts scheduled__2025-03-07T23:00:00+00:00 [running]> on host 45b38884c639
[2025-04-28T18:17:53.035+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='longnv' AIRFLOW_CTX_DAG_ID='gen_lxw_fts' AIRFLOW_CTX_TASK_ID='mateiralize_fts' AIRFLOW_CTX_EXECUTION_DATE='2025-03-07T23:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-07T23:00:00+00:00'
[2025-04-28T18:17:53.036+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-28T18:17:53.038+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '\n        python /opt/***/external_scripts/step_3_1_fs_register_table.py\n        ']
[2025-04-28T18:17:53.053+0000] {subprocess.py:86} INFO - Output:
[2025-04-28T18:18:00.376+0000] {subprocess.py:93} INFO - :: loading settings :: url = jar:file:/opt/spark/jars/ivy-2.5.1.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2025-04-28T18:18:00.485+0000] {subprocess.py:93} INFO - Ivy Default Cache set to: /home/***/.ivy2/cache
[2025-04-28T18:18:00.488+0000] {subprocess.py:93} INFO - The jars for the packages stored in: /home/***/.ivy2/jars
[2025-04-28T18:18:00.491+0000] {subprocess.py:93} INFO - org.apache.spark#spark-sql-kafka-0-10_2.12 added as a dependency
[2025-04-28T18:18:00.492+0000] {subprocess.py:93} INFO - io.delta#delta-core_2.12 added as a dependency
[2025-04-28T18:18:00.492+0000] {subprocess.py:93} INFO - org.apache.hadoop#hadoop-aws added as a dependency
[2025-04-28T18:18:00.492+0000] {subprocess.py:93} INFO - com.amazonaws#aws-java-sdk-bundle added as a dependency
[2025-04-28T18:18:00.493+0000] {subprocess.py:93} INFO - org.postgresql#postgresql added as a dependency
[2025-04-28T18:18:00.493+0000] {subprocess.py:93} INFO - :: resolving dependencies :: org.apache.spark#spark-submit-parent-bbca2150-61c9-4059-9986-84a0568151f1;1.0
[2025-04-28T18:18:00.494+0000] {subprocess.py:93} INFO - 	confs: [default]
[2025-04-28T18:18:00.693+0000] {subprocess.py:93} INFO - 	found org.apache.spark#spark-sql-kafka-0-10_2.12;3.4.1 in central
[2025-04-28T18:18:00.998+0000] {subprocess.py:93} INFO - 	found org.apache.spark#spark-token-provider-kafka-0-10_2.12;3.4.1 in central
[2025-04-28T18:18:01.084+0000] {subprocess.py:93} INFO - 	found org.apache.kafka#kafka-clients;3.3.2 in central
[2025-04-28T18:18:01.128+0000] {subprocess.py:93} INFO - 	found org.lz4#lz4-java;1.8.0 in central
[2025-04-28T18:18:01.165+0000] {subprocess.py:93} INFO - 	found org.xerial.snappy#snappy-java;1.1.10.1 in central
[2025-04-28T18:18:01.189+0000] {subprocess.py:93} INFO - 	found org.slf4j#slf4j-api;2.0.6 in central
[2025-04-28T18:18:01.257+0000] {subprocess.py:93} INFO - 	found org.apache.hadoop#hadoop-client-runtime;3.3.4 in central
[2025-04-28T18:18:01.350+0000] {subprocess.py:93} INFO - 	found org.apache.hadoop#hadoop-client-api;3.3.4 in central
[2025-04-28T18:18:01.389+0000] {subprocess.py:93} INFO - 	found commons-logging#commons-logging;1.1.3 in central
[2025-04-28T18:18:01.409+0000] {subprocess.py:93} INFO - 	found com.google.code.findbugs#jsr305;3.0.0 in central
[2025-04-28T18:18:01.418+0000] {subprocess.py:93} INFO - 	found org.apache.commons#commons-pool2;2.11.1 in central
[2025-04-28T18:18:01.434+0000] {subprocess.py:93} INFO - 	found io.delta#delta-core_2.12;2.4.0 in central
[2025-04-28T18:18:01.448+0000] {subprocess.py:93} INFO - 	found io.delta#delta-storage;2.4.0 in central
[2025-04-28T18:18:01.456+0000] {subprocess.py:93} INFO - 	found org.antlr#antlr4-runtime;4.9.3 in central
[2025-04-28T18:18:01.490+0000] {subprocess.py:93} INFO - 	found org.apache.hadoop#hadoop-aws;3.3.2 in central
[2025-04-28T18:18:01.503+0000] {subprocess.py:93} INFO - 	found org.wildfly.openssl#wildfly-openssl;1.0.7.Final in central
[2025-04-28T18:18:01.513+0000] {subprocess.py:93} INFO - 	found com.amazonaws#aws-java-sdk-bundle;1.12.261 in central
[2025-04-28T18:18:01.531+0000] {subprocess.py:93} INFO - 	found org.postgresql#postgresql;42.5.1 in central
[2025-04-28T18:18:01.540+0000] {subprocess.py:93} INFO - 	found org.checkerframework#checker-qual;3.5.0 in central
[2025-04-28T18:18:01.576+0000] {subprocess.py:93} INFO - :: resolution report :: resolve 1060ms :: artifacts dl 18ms
[2025-04-28T18:18:01.577+0000] {subprocess.py:93} INFO - 	:: modules in use:
[2025-04-28T18:18:01.578+0000] {subprocess.py:93} INFO - 	com.amazonaws#aws-java-sdk-bundle;1.12.261 from central in [default]
[2025-04-28T18:18:01.584+0000] {subprocess.py:93} INFO - 	com.google.code.findbugs#jsr305;3.0.0 from central in [default]
[2025-04-28T18:18:01.585+0000] {subprocess.py:93} INFO - 	commons-logging#commons-logging;1.1.3 from central in [default]
[2025-04-28T18:18:01.585+0000] {subprocess.py:93} INFO - 	io.delta#delta-core_2.12;2.4.0 from central in [default]
[2025-04-28T18:18:01.586+0000] {subprocess.py:93} INFO - 	io.delta#delta-storage;2.4.0 from central in [default]
[2025-04-28T18:18:01.586+0000] {subprocess.py:93} INFO - 	org.antlr#antlr4-runtime;4.9.3 from central in [default]
[2025-04-28T18:18:01.587+0000] {subprocess.py:93} INFO - 	org.apache.commons#commons-pool2;2.11.1 from central in [default]
[2025-04-28T18:18:01.592+0000] {subprocess.py:93} INFO - 	org.apache.hadoop#hadoop-aws;3.3.2 from central in [default]
[2025-04-28T18:18:01.592+0000] {subprocess.py:93} INFO - 	org.apache.hadoop#hadoop-client-api;3.3.4 from central in [default]
[2025-04-28T18:18:01.593+0000] {subprocess.py:93} INFO - 	org.apache.hadoop#hadoop-client-runtime;3.3.4 from central in [default]
[2025-04-28T18:18:01.593+0000] {subprocess.py:93} INFO - 	org.apache.kafka#kafka-clients;3.3.2 from central in [default]
[2025-04-28T18:18:01.596+0000] {subprocess.py:93} INFO - 	org.apache.spark#spark-sql-kafka-0-10_2.12;3.4.1 from central in [default]
[2025-04-28T18:18:01.596+0000] {subprocess.py:93} INFO - 	org.apache.spark#spark-token-provider-kafka-0-10_2.12;3.4.1 from central in [default]
[2025-04-28T18:18:01.596+0000] {subprocess.py:93} INFO - 	org.checkerframework#checker-qual;3.5.0 from central in [default]
[2025-04-28T18:18:01.598+0000] {subprocess.py:93} INFO - 	org.lz4#lz4-java;1.8.0 from central in [default]
[2025-04-28T18:18:01.599+0000] {subprocess.py:93} INFO - 	org.postgresql#postgresql;42.5.1 from central in [default]
[2025-04-28T18:18:01.602+0000] {subprocess.py:93} INFO - 	org.slf4j#slf4j-api;2.0.6 from central in [default]
[2025-04-28T18:18:01.602+0000] {subprocess.py:93} INFO - 	org.wildfly.openssl#wildfly-openssl;1.0.7.Final from central in [default]
[2025-04-28T18:18:01.602+0000] {subprocess.py:93} INFO - 	org.xerial.snappy#snappy-java;1.1.10.1 from central in [default]
[2025-04-28T18:18:01.603+0000] {subprocess.py:93} INFO - 	:: evicted modules:
[2025-04-28T18:18:01.610+0000] {subprocess.py:93} INFO - 	com.amazonaws#aws-java-sdk-bundle;1.11.1026 by [com.amazonaws#aws-java-sdk-bundle;1.12.261] in [default]
[2025-04-28T18:18:01.610+0000] {subprocess.py:93} INFO - 	---------------------------------------------------------------------
[2025-04-28T18:18:01.610+0000] {subprocess.py:93} INFO - 	|                  |            modules            ||   artifacts   |
[2025-04-28T18:18:01.611+0000] {subprocess.py:93} INFO - 	|       conf       | number| search|dwnlded|evicted|| number|dwnlded|
[2025-04-28T18:18:01.611+0000] {subprocess.py:93} INFO - 	---------------------------------------------------------------------
[2025-04-28T18:18:01.614+0000] {subprocess.py:93} INFO - 	|      default     |   20  |   0   |   0   |   1   ||   19  |   0   |
[2025-04-28T18:18:01.618+0000] {subprocess.py:93} INFO - 	---------------------------------------------------------------------
[2025-04-28T18:18:01.620+0000] {subprocess.py:93} INFO - :: retrieving :: org.apache.spark#spark-submit-parent-bbca2150-61c9-4059-9986-84a0568151f1
[2025-04-28T18:18:01.624+0000] {subprocess.py:93} INFO - 	confs: [default]
[2025-04-28T18:18:01.624+0000] {subprocess.py:93} INFO - 	0 artifacts copied, 19 already retrieved (0kB/12ms)
[2025-04-28T18:18:02.035+0000] {subprocess.py:93} INFO - 25/04/28 18:18:02 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-04-28T18:18:02.450+0000] {subprocess.py:93} INFO - Setting default log level to "WARN".
[2025-04-28T18:18:02.450+0000] {subprocess.py:93} INFO - To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).
[2025-04-28T18:18:03.016+0000] {local_task_job_runner.py:302} WARNING - State of this instance has been externally set to up_for_retry. Terminating instance.
[2025-04-28T18:18:03.017+0000] {process_utils.py:131} INFO - Sending 15 to group 9248. PIDs of all processes in the group: [9249, 9250, 9292, 9248]
[2025-04-28T18:18:03.018+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 9248
[2025-04-28T18:18:03.019+0000] {taskinstance.py:2451} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-04-28T18:18:03.025+0000] {subprocess.py:104} INFO - Sending SIGTERM signal to process group
[2025-04-28T18:18:03.063+0000] {taskinstance.py:2699} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 203, in execute
    result = self.subprocess_hook.run_command(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/subprocess.py", line 91, in run_command
    for raw_line in iter(self.sub_process.stdout.readline, b""):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2453, in signal_handler
    raise AirflowException("Task received SIGTERM signal")
airflow.exceptions.AirflowException: Task received SIGTERM signal
[2025-04-28T18:18:03.085+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=9250, status='terminated', started='18:17:52') (9250) terminated with exit code None
[2025-04-28T18:18:03.086+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=gen_lxw_fts, task_id=mateiralize_fts, execution_date=20250307T230000, start_date=20250428T181752, end_date=20250428T181803
[2025-04-28T18:18:03.118+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 285 for task mateiralize_fts (Task received SIGTERM signal; 9248)
[2025-04-28T18:18:03.144+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=9248, status='terminated', exitcode=1, started='18:17:52') (9248) terminated with exit code 1
[2025-04-28T18:18:03.420+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=9249, status='terminated', started='18:17:52') (9249) terminated with exit code None
[2025-04-28T18:18:03.563+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=9292, status='terminated', started='18:17:56') (9292) terminated with exit code None
